### PR TITLE
Change v1 api routes for non-OCamlTypes, non-ocamlCompatible responses 

### PR DESCRIPTION
**Pull request by pbiggar**
This is best read in commit order.

This takes the v1 apis (which exist but are not used) and adapts them to not use any of the OCaml stuff (either the OCamlTypes or the OCamlCompatible json serializer).

I first made copies of everything with the old formats, so that we'd have a diff to see the change from the old format.

The next step is to switch the client over to the new routes.

### CODE REVIEW

File: fsharp-backend/src/ApiServer/Api/APIInitialLoad.fs [hunk 1]
```diff
@@ -22,123 +22,248 @@ module Canvas = LibBackend.Canvas
...
-
-    return result
-  }
+  type ApiStaticDeploy =
+    { deploy_hash : string
+      url : string
+      last_update : NodaTime.Instant
+      status : SA.DeployStatus }
+
+  let toApiStaticDeploys (d : SA.StaticDeploy) : ApiStaticDeploy =
```
Comment (StachuDotNet) [inline fsharp-backend/src/ApiServer/Api/APIInitialLoad.fs:39]: probably `toApiStaticDeploy`?

File: fsharp-backend/src/ApiServer/ClientTypes.fs [hunk 2]
```diff
@@ -0,0 +1,167 @@
+/// Types used in many APIs
+module ApiServer.ClientTypes
```
Comment (StachuDotNet) [inline fsharp-backend/src/ApiServer/ClientTypes.fs:2]: Much clearer, great!

**Review by StachuDotNet — CHANGES_REQUESTED** [review]
minor questions/notes, but overall looks good

File: fsharp-backend/src/ApiServer/Api/APIInitialLoad.fs (hunk 1 above)
Comment (pbiggar) [reply fsharp-backend/src/ApiServer/Api/APIInitialLoad.fs:39]: this was a straight copy, i don't want to change anything unnecessary

File: fsharp-backend/src/ApiServer/Api/APIOps.fs [hunk 3]
```diff
@@ -30,136 +30,280 @@ open LibBackend.Db
...
+                  // https://github.com/darklang/dark/issues/3675 for discussion.
+                  None
+            Option.map (fun (tl, deleted) -> (tlid, oplists, tl, deleted)) tlPair)
+          |> C.saveTLIDs canvasInfo
+
+
+      t.next "send-ops-to-pusher"
+      let event =
+        // To make this work with prodclone, we might want to have it specify
+        // more ... else people's prodclones will stomp on each other ...
```
[the remark this answers is not in the document]
Comment (pbiggar) [reply fsharp-backend/src/ApiServer/Api/APIOps.fs:145]: same, not my comment

**Pull request merged**